src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import "../../style/common.scss";
+import "../../pages/Introduce/Introduce.scss";
+import { Link } from "react-router-dom";
+
+const Introduce = () => {
+  return (
+    <div style={boxStyle} className="box">
```

Comment:
```suggestion
    <div className="introduce">
```

인라인 스타일은 지양해주세요!

컴포넌트의 가장 부모태그에는 컴포넌트의 이름과 동일하게 구현해주세요!